src/functional/ffront/symbol_makers.py

```diff
@@ -168,22 +168,16 @@ def make_symbol_type_from_typing(
 def make_symbol_from_value(
     name: str, value: Any, namespace: foast.Namespace, location: SourceLocation
 ) -> foast.Symbol:
-    if not isinstance(value, type) or type(value).__module__ != "typing":
-        value = typingx.get_typing(value, annotate_callable_kwargs=True)
-
-    symbol_type = make_symbol_type_from_typing(value)
-
-    if isinstance(symbol_type, common_types.DataType):
-        return foast.DataSymbol(id=name, type=symbol_type, namespace=namespace, location=location)
-    elif isinstance(symbol_type, common_types.FunctionType):
-        return foast.Function(
-            id=name,
-            type=symbol_type,
-            namespace=namespace,
-            location=location,
-        )
-    elif isinstance(symbol_type, common_types.OffsetType):
-        return foast.OffsetSymbol(id=name, type=symbol_type, namespace=namespace, location=location)
+    """Make a symbol node from a python value."""
+    assert not isinstance(value, type) and type(value).__module__ != "typing"
```

Comment:
If passing types-like args is forbidden, I think it'd better to raise a `ValueError` exception with a meaningful message.